### ISSUE TITLE
idna crate locked to 0.5.0 in 1.18.1 release, SEC 

### ISSUE THREAD

**Issue by vancejc-mt** [Keats/validator#362]
A new SEC was released which suggests that `idna` be upgraded to `1.0.0`.  While this has been updated in `master`, the latest release (`1.18.1`) still has this locked to `0.5.0`.

Any chance we could get a release after `1.18.1` which updates the `idna` version?

https://rustsec.org/advisories/RUSTSEC-2024-0421

**Comment by Keats** [comment]
Latest release is 0.19 which uses idna 1.0

**Comment by Dentrax** [comment]
Hey @Keats, it seems latest release is still 0.18.1: https://github.com/Keats/validator/tags
Correct me if I'm missing something. Thanks!

**Comment by Keats** [comment]
Yeah I'm not sure how to use the tags in this repo since there are 2 crates so I don't really use them too much
